script/genkeys

```diff
@@ -13,13 +13,23 @@ KEYMASTER="docker run --volume ${ROOT}/certificates:/certificates --rm cloudpipe
 if [ ! -f "${ROOT}/certificates/password" ]; then
   echo ">> creating a random password in ${ROOT}/certificates/password."
   touch ${ROOT}/certificates/password
-  chmod 600 ${ROOT}/certificates/password
+  chmod 640 ${ROOT}/certificates/password
   # "If the same pathname argument is supplied to -passin and -passout arguments then the first
   # line will be used for the input password and the next line for the output password."
-  cat /dev/random | head -c 128 | base64 > ${ROOT}/certificates/password
+  cat /dev/urandom | head -c 128 | base64 > ${ROOT}/certificates/password
```

Comment:
You do realize that's the same document I referenced in my commit message? :)

My chief assertion is that iff the genkey script is intended for CI/continuous development usage, than urandom is functionally equivalent, and more appropriate for a continuous workflow.

If not, then `/dev/random` could maybe make sense -- but not even then I'd think.  In that situation wouldn't we be talking production where you should probably be doing the PKI bits separately from this anyway?